python/runtime/cudaq/algorithms/py_draw.cpp

```diff
@@ -17,7 +17,7 @@
 
 namespace cudaq {
 
-namespace details {
+namespace contrib {
```

Comment:
I think this change was accidental? getKernelLaunchParameters does not belong in contrib.
Edit: I guess not accidental. We should still keep it in details.